tests/integration_tests/test_calibration/test_eeprom.py

```diff
@@ -28,27 +30,36 @@ def test__byte_write_register(data, address, expected, eeprom):
     addrx = EdgePiMemoryInfo.USER_SPACE_START_BYTE.value + address
     # pylint: disable=protected-access
     eeprom._EdgePiEEPROM__byte_write_register(addrx, data)
-    time.sleep(0.001)
+    time.sleep(0.002)
```

Comment:
@sjpark608 it is a good idea to add a comment here. I think we might have talked about this in the code review as well.

Reply:
yes we've talked about this in the code review. I added the comment in one place, but I will add them to everywhere I use it. I'm gonna open a ticket as well

Reply:
> What happens if you don't sleep? Does that method for the EEPROM execute asynchronously? Otherwise, wouldn't it wait for that method to finish before moving on?

The I2C driver fails for some reason when there's no sleep delay. It will throw an OSError and fails to perform I2C transaction.